travel_platform/api/izi_travel/izi_travel_client.py

```diff
@@ -37,7 +37,7 @@ def _check_response_status(self, response):
                 response.status_code, response.content))
 
     def _prepare_image_url(self, content_provider_uuid,
-                           image_uuid, image_size='800x600'):
+                           image_uuid, image_size='480x360'):
```

Comment:
I think it's pointless but i can just remove default image size anytime and get original pics.